lib/bundler/cli/gem.rb

```diff
@@ -139,6 +139,10 @@ def run
 
       # Open gemspec in editor
       open_editor(options["edit"], target.join("#{name}.gemspec")) if options[:edit]
+    rescue SystemCallError => e
```

Comment:
@segiddins I have just noticed that here https://github.com/bundler/bundler/blob/master/lib/bundler/shared_helpers.rb#L120 when `Errno::EEXIST` happen a generic `RuntimeError` is thrown. Are you OK if I create a custom exception for that?